src/main/webapp/WEB-INF/views/authentication/register.jsp

```diff
@@ -0,0 +1,39 @@
+<%@ page contentType="text/html;charset=UTF-8" language="java" %>

```

Comment:
as @KseniiaMakarova commented earlier, let's try to minimize the breaking changes in your PR and use the old `drivers/add` jsp and URL. 
Overall it is better to not change the URLs because some other services may already be integrated with ours or a user might get used to the old ones, and it is always nice to keep it as it is